python/py-onnx/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        onnx onnx 1.6.0 v
+revision            0
+name                py-${github.project}
+categories-append   science
+fetch.type          git
+
+platforms           darwin
+supported_archs     noarch
+
+license             BSD
+
+maintainers         nomaintainer
+
+description         Open Neural Network Exchange
+
+long_description    Open Neural Network Exchange (ONNX) is an open\
+                    ecosystem that empowers AI developers to choose\
+                    the right tools as their project evolves. ONNX\
+                    provides an open source format for AI models, both\
+                    deep learning and traditional ML. It defines an\
+                    extensible computation graph model, as well as\
+                    definitions of built-in operators and standard\
+                    data types. Currently we focus on the capabilities\
+                    needed for inferencing (scoring).
+
+homepage            https://onnx.ai/
+
+# Support python versions.
+python.versions     27 35 36 37
+
+compiler.cxx_standard 2011
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+        port:git \
+        port:cctools \
+        path:bin/cmake:cmake \
+        port:py${python.version}-protobuf3 \
+        port:py${python.version}-pytest-runner
+
+    depends_lib-append \
+        port:protobuf-c \
+        port:py${python.version}-future \
+        port:py${python.version}-numpy \
+        port:py${python.version}-setuptools \
+        port:py${python.version}-six
+
+    if { ${python.version} < 35 } {
+        depends_lib-append \
+            port:py${python.version}-typing \
+    }
+
+    if { ${python.version} >= 37 } {
+        depends_lib-append \
+            port:py${python.version}-typing_extensions
+    }
+
+    depends_test-append \
+        port:py${python.version}-nbval \
+        port:py${python.version}-pytest \
+        port:py${python.version}-tabulate
+
+    post-fetch {
+        system -W ${worksrcpath} "git submodule update --init --recursive"
+    }
+
```

Comment:
You should likely fix the issues below reported by the build system about ```pybind11```. I am not sure even if you need/want to do the ```git submodule``` command at all to get the third-party extensions, why not use MacPort's ```py-pybind11``` port?
 
```
CMake Warning at CMakeLists.txt:394 (find_package):
  By not providing "Findpybind11.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "pybind11",
  but CMake did not find one.

  Could not find a package configuration file provided by "pybind11"
  (requested version 2.2) with any of the following names:

    pybind11Config.cmake
    pybind11-config.cmake

  Add the installation prefix of "pybind11" to CMAKE_PREFIX_PATH or set
  "pybind11_DIR" to a directory containing one of the above files.  If
  "pybind11" provides a separate development package or SDK, be sure it has
  been installed.
```

Reply:
Thanks done. It works too.